spec/controllers/items_controller.rb

```diff
@@ -1,6 +1,52 @@
 require 'rails_helper'
 
 RSpec.describe ItemsController, type: :controller do
+
+  describe '#new' do
+    let!(:order) { create(:order) }
+    before { get :new, params: { id: order.id } }
+    describe 'successful response' do
+      it { expect(response).to be_successful }
+      it { expect(response).to render_template('new') }
+    end
+    context 'item' do
+      it { expect(assigns(:item)).to be_a(Item) }
+      it { expect(assigns(:item).persisted?).to eq(false) }
+    end
+  end
+
+  describe '#create' do
+    login_user
+    let!(:order) { create(:order) }
+    let(:valid_attributes) { { item: {food: 'Jedzenie', cost: 12.50, order_id: order.id} } }
+    let(:invalid_attributes) { { item: {food: '', cost: nil} } }
+
+    context 'valid params' do
+      subject { post :create, params: valid_attributes }
+
+      it 'should redirect to item' do
+        expect(subject).to redirect_to(item_path 1 )
+      end
+      it 'should redirect with notice' do
+        subject
+        expect(flash[:notice]).to be_present
+      end
+      it 'should create new item' do
+        expect{subject}.to change{ Item.count }.by(1)
+      end
+    end
+
+    # context 'invalid params' do
```

Comment:
Why is this commented out?

Reply:
I forgot about these tests. It should be fixed now.